### DISCUSSION TITLE
Trying to find a demo

### DISCUSSION THREAD

**Discussion by cvillalba** [microrealestate/microrealestate#114] [Q&A] [answered]
Hello everyone

Is there a demo available to have a look at the features and see if we can use this project?
Thanks for your answer in advance :)

**Comment by cvillalba** [comment]
@camelaissani any link for a demo available, thanks for your help 😄

**Reply by camelaissani** [reply]
Hi @cvillalba not yet but I'm working on that. I hope I'll be able to do that through #106.
Actually I need to be able to host the app in a VM.

**Reply by cvillalba** [reply]
@camelaissani thanks for your answer, I was actually able to run it on a VM (dreamhost) but I couldn't create an account to login to the system. I'll be looking forward to the availability of the Demo, I'll have a look to see if I can help with something in the project.

**Reply by camelaissani** [reply]
I may help you to understand what is missing if you can give me a link to access the UI.
I suppose the `gateway` (authenticator endpoint) is not accessible from the UI

**Reply by cvillalba** [reply]
@camelaissani sorry my late answer, I had to get the whole instance back up again. here I am http://208.113.128.196:8080/landlord/signup

**Reply by cvillalba** [reply]
I think I see the issue, everything is been redirected to localhost, failing the requests

**Reply by cvillalba** [reply]
Don't know how to fix this though

**Reply by cvillalba** [reply]
@camelaissani can I get you help on this please?

**Reply by camelaissani** [reply, marked as answer]
@cvillalba in you current installation the frontend cannot reach the backend (gateway).
Can you test that ?
1. Stop the application `mre stop`
2. Update the `.env` file at the root of the project to set this environment variable `GATEWAY_URL=http://208.113.128.196:8080/api/v2`
3. Rebuild the application `mre build`
4. Restart the application `mre start`

**Reply by cvillalba** [reply]
Thanks for your help
